iopipe/contrib/eventinfo/event_types.py

```diff
@@ -21,9 +21,13 @@ def collect(self):
             return collect_all_keys(self.event, '@iopipe/event-info.%s' % self.type, self.exclude_keys)
         event_info = {}
         for key in self.keys:
-            value = get_value(self.event, key)
+            if isinstance(key, tuple):
```

Comment:
What's changing that we need to check that these are tuples?

Reply:
We map one key to another, so `keys` can now be a list of strings or a list of tuples. If it's a list of tuples, it's a map from `old_key` to `new_key`.

Reply:
Why can `keys` now be of two different types? (List of strings or list of tuples)

Reply:
Because we map serveless lambda proxy events into api gateway events. See the equivalent JS implementation here:
https://github.com/iopipe/iopipe-js-event-info/commit/2033c3e66976de1855a5746f33c745bf76492b03